README.md

```diff
@@ -12,14 +12,37 @@ The simplest way to forward all your container's log to Rapid7 InsightOps is to
 run this repository as a container, with:
 
 ```sh
-docker run -v /var/run/docker.sock:/var/run/docker.sock rapid7/r7insight_docker -t <TOKEN> -r <REGION> -j -a host=`uname -n`
+docker run -v /var/run/docker.sock:/var/run/docker.sock \
```

Comment:
True, I found that out first hand.
I'd say stick with this, since we're supposed to be documenting and it's function is informative. This is more readable.